webrtc.html

```diff
@@ -685,7 +685,7 @@ <h4><dfn>RTCSignalingState</dfn> Enum</h4>
               </tr>
               <tr>
                 <td><dfn data-idl><code>stable</code></dfn></td>
-                <td>There is no offer&shy;answer exchange in progress. This is
+                <td>There is no offer/answer exchange in progress. This is
```

Comment:
while picking nits, can we add a comma between "initial state" and "in which case" below?